src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -196,6 +204,17 @@ export default {
 		unreservedAmount() {
 			return this.loan?.unreservedAmount ?? '';
 		},
+		amountLeft() {
```

Comment:
Looks like we still need to check the `unreservedAmount` before returning 5/25 in this method.

Reply:
You are so fast! 😛